run_bonsai.sh

```diff
@@ -0,0 +1,9 @@
+export BONSAI_API_KEY="1234"
+export BONSAI_API_URL="https://api.bonsai.xyz/"
+# https://dev.risczero.com/api/blockchain-integration/contracts/verifier
+# RiscZeroGroth16Verifier.sol Sepolia	0x83C2e9CD64B2A16D3908E94C7654f3864212E2F8
+export GROTH16_VERIFIER_ADDRESS="0x83C2e9CD64B2A16D3908E94C7654f3864212E2F8"
```

Comment:
I guess the downside of making these environment variables is that a restart is needed to test different networks. Not the worst thing, but making these normal inputs like the rest of the proof inputs may be worth it to keep things consistent and flexible without having to do restarts.

Reading the BONSAI API and URL from environment variables makes a lot of sense though.

Reply:
Ah right, good point that it just needs to point so some verifier, it doesn't really matter where it is. Then either environment value or the struct you proposed good for me!

Reply:
Ok, let's keep that to avoid more code changes :-)